[Repository] Add .github folder with contributing file (#3362)
lukesampson/scoop: .github/ISSUE_TEMPLATE/bug.md

---
name: Bug

> about: Problem which make it impossible to use scoop.

Reviewer: ```suggestion
A problem which makes it impossible to use Scoop.
```

labels: bug
---

<!-- Describe your problem: -->

> <!-- Add screenshot, reproducible steps or terminal output: -->

Reviewer: ```suggestion
<!-- Add a screenshot, reproducible steps or terminal output: -->
```

- **Branch**: master/develop <!-- `scoop config SCOOP_BRANCH | clip` -->
- **Powershell Version**: <!-- Use: `$PSVersionTable.PSVersion.ToString() | clip` -->

> <!-- Are you under special environment? Do you use proxy? Can you provide other relevant information which could help with problem identifying? -->

Reviewer: ```suggestion
<!-- Are you under a special environment? Do you use a proxy? Can you provide other relevant information which could help with problem identifying? -->
```
